server/channels/db/migrations/postgres/000129_add_property_groups.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS PropertyGroups (
+	ID varchar(26) PRIMARY KEY,
+	Name varchar(64) NOT NULL,
+	UNIQUE(Name)
+);
```

Comment:
nitpick: we don't allow for deleting/modifying groups?

I don't think it is needed for this phase, but something to consider

Reply:
Not at this phase of the feature, we'd have to define the expectations on what should happen when deleting a group, what features would require that, etc.